magento/pwa-studio
# [Docs] Release notes 8.0.0

## Description

Updates the compatibility table and changelog with information about the 8.0.0 release.

## Related Issue





Closes PWA-927

## Acceptance 

@cherdman 
@ericerway (optional)

### Verification Stakeholders

Any developer

@cherdman 
@ericerway (optional)

### Specification



### Verification Steps

1. Open CHANGELOG.md
1. Verify release information is accurate

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
